src/Symfony/Component/Routing/Tests/Fixtures/dumper/static_url_matcher3.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+// This file has been auto-generated by the Symfony Routing Component.
+
+return [
+    [
+        [
+            null,
+            [
+                null,
+                [
+                    '/rootprefix',
+                    [
+                        [
+                            'path',
+                            '/rootprefix/test'
+                        ],
+                        null,
+                        [],
+                        [],
+                        [
+                            '_route' => 'static'
+                        ]
+                    ],
+                    [
+                        [
+                            'match',
+                            '#^/rootprefix/(?P<var>[^/]++)$#s',
```

Comment:
Maybe not entirely related to this PR, but would it also be an idea to optimize routes away that can't ever be matched? For example, I register a bunch of routes which are exposed via a bundle, but they link to an external application. Those routes will show up in the URL matcher, while I really only need them in the generator.

Example:
```
// frontend_routing.hostnetnl.search
if ('/zoeken' === $pathinfo) {
    return $this->mergeDefaults(array_replace($hostMatches, array('_route' => 'frontend_routing.hostnetnl.search')), array ());
}
```

The reason I mention this, is because while being a URL matcher fixture, it doesn't seem like an actual fixture that could be called if used in a real world application. Currently not too much overhead because they are loaded last in my scenario, but it's still 51 routes in the current version of my bundle. This being added to an array, but _never_ used, it feels like an optimization would be very welcome!

Reply:
@iltar we should certainly not do that in this PR as this is a separate concern. The current PR is big enough :)